### PR TITLE
A complete beginner guide on installing flask for beginners

### PR DESCRIPTION
**Pull request by corpsgeek**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by corpsgeek** [comment]
This pull request is linked to the issue: https://github.com/section-io/engineering-education/issues/717

**Comment by hectorkambow** [comment]
Plagiarism check done

**Review by adrianmurage — CHANGES_REQUESTED** [review]
@corpsgeek
Great article. I took the liberty to edit out a few things.
Kindly make a point of going through the commits I've made and note the things I changed for future contributions.

Avoid commentary and go to the point whenever possible.
@ me when you've made the changes I've commented on.

File: articles/a-complete-guide-on-installing-flask-for-beginners/index.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+~/Desktop$ mkdir flask_website
+~/Desktop$ ls
+flask_website
+~/Desktop$ cd flask_website
+~/Desktop/flask_website$ python3 -m venv env
+~/Desktop/flask_website$ ls
+env
+```
+
+By following the three steps above, we have our virtual environment for our Flask app ready, next, we activate it.
```
[the remark this answers is not in the document]
Comment (corpsgeek) [reply articles/a-complete-guide-on-installing-flask-for-beginners/index.md:67]: Alright sir, I'll work on that right away.

**Comment by corpsgeek** [comment]
@adrianmurage as a new contributor, how do I go about making the recommended changes as requested in the index.MD file, and submitting back here for review? A step-by-step approach would be appreciated. Thank you

**Comment by corpsgeek** [comment]
@adrianmurage I've made the changes as requested. I also went through some of the changes you made in the markdown and updated that for other sections of the article.

**Review by adrianmurage — APPROVED** [review]

**Comment by adrianmurage** [comment]
@hectorkambow 
This article is ready for final review

**Comment by adrianmurage** [comment]
@corpsgeek 
Great job with this article. Looking forward to seeing more contributions from you in the near future : )

**Comment by corpsgeek** [comment]
Alright @adrianmurage, definitely more to come. Its a pleasure working with you.

**Comment by hectorkambow** [comment]
@corpsgeek 👋  Thank you v much for contributing your 1st article - and Welcome to the EngEd Program. 

Please be sure to go over any extra edits I may have done - to help incorporate those into future PRs. 

Just as a friendly FYI - articles that required over one round of revision will receive a [tiered payout.](https://github.com/section-io/engineering-education/blob/master/new_contributors/CONTRIBUTING.md#review-approval--payment-processes)

Please also be sure to go over our [Resources Page](https://github.com/section-io/engineering-education/blob/master/new_contributors/resources-page.md), where we have laid out valuable resources meant to help improve our community members' overall technical writing abilities. 

cc @adrianmurage 

Looking forward to going over more amazing content 🚀

**Pull request merged**